dbt/adapters/sqlserver/connections.py

```diff
@@ -49,8 +34,8 @@ class SQLServerCredentials(Credentials):
     # "sql", "ActiveDirectoryPassword" or "ActiveDirectoryInteractive", or
     # "ServicePrincipal"
     authentication: Optional[str] = "sql"
-    encrypt: Optional[bool] = False
-    trust_cert: Optional[bool] = False
+    encrypt: Optional[bool] = True
```

Comment:
This code comes from [dbt-synapse](https://github.com/dbt-msft/dbt-synapse), there `encrypt` and `trust_cert` are set to True.

Reply:
This was changed to not break connections for all users already using dbt-sqlserver. The `encrypt: Optional[bool] = True` would potentially break connections for people when upgrading dbt-sqlserver.

Reply:
Do we need to set it back here to the `dbt-sqlserver` settings?

Reply:
🤷  probably? I'm leaning towards publishing a breaking change for existing users to make sure that future users are using security best practices. Microsoft themselves have made `Encrypt=True` to be the defaults as well...

Reply:
To limit the scope of this PR, I have set them back to the original defaults, this can be done in a separate PR.